server/src/api/modules/auth.js

```diff
@@ -11,11 +11,15 @@ export const register = (req, res) => {
       if (user) {
         return res.status(404).send("Username already exists")
       }
-      let newUser = new User({ username, password, email })
+      let newUser = new User({
+        username,
+        password,
+        email
+      })
       newUser
         .save()
         .then(() => {
-          res.status(201).send("success")
+          res.status(201).send()
```

Comment:
If you'd prefer not to send a confirmation message perhaps change to `res.sendStatus(201)`